test/ICEDdemo.py

```diff
@@ -139,25 +139,26 @@ def DVFSMap(kernels,df):
                         dataS.append(int(outputLine.split("histogram 100% tile DVFS frequency ratio: ")[1]))
                     
     except eventlet.timeout.Timeout:
+        # print("Skipping a specific config for kernel: ", kernel, "Because it runs more than", timeOutSet/60 , "minute(s).")
         dataS = [0]*(dictColumn)
-        print("Skipping a specific config for kernel: ", kernels)
 
     df.loc[len(df.index)] = dataS
 
 
-def DVFSGen(kernels,df):
+def DVFSGen(kernel, df):
     """
     This is a func gain DFG information only for showTableI().
 
-    Parameters: kernel name, df array to collect mapping information of kernels
+    Parameters: file name of a kernel, df array to collect mapping information of the kernel
 
     Returns: NULL
     """
     getMapCommand = "opt-12 -load ../build/src/libmapperPass.so -mapperPass kernel.bc"
     genMapProc = subprocess.Popen([getMapCommand, "-u"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
     dataS = []    # for get results from subprocess and output to pandas
-    kernelsSource = (kernels.split("."))[0]
+    kernelsSource = (kernel.split("."))[0]
     dataS.append(kernelsSource)
+    dataSHead = 4   # the first 4 element of dataS is not empty: kernelsSource, DFG node count, DFG edge count, RecMII
```

Comment:
nit: Rename to `DataSHead = 4` as it is a constant.

Reply:
Thanks for paying attention to my coding style.